docs/source/reference/comparison.rst

```diff
@@ -0,0 +1,178 @@
+.. _sky-compare:
+
+Comparing SkyPilot with other systems
+=====================================
+
+We are often asked "How does SkyPilot compare with XYZ?". Providing an unbiased and up-to date answer for such questions is not easy, especially when the differences may be qualitative.
+
+This page tries to provide a comparison of SkyPilot with other systems, focusing on the unique features of SkyPilot. We welcome feedback and contributions to this page.
+
+
+SkyPilot vs Vanilla Kubernetes
+------------------------------
+
+Kubernetes is a powerful system for managing containerized applications. :ref:`Using SkyPilot to access your Kubernetes cluster <kubernetes-overview>` boosts developer productivity and allows you to scale your infra beyond a single Kubernetes cluster.
```

Comment:
Can we show a architecture diagram for showing how SkyPilot relates to kubernetes clusters? For example, a user interacts with SkyPilot and SkyPilot sends the requests to underlying Kubernetes.

Reply:
Good idea - added an architecture figure. wdyt?

Reply:
LGTM.